

class TaskAlreadyExistsError(Exception):
    pass


class NoSuchTaskError(Exception):
    pass